setup.py

```diff
@@ -52,12 +52,11 @@ def run(self):
         print("=> Building mod files ...")
 
         if platform.system() == 'Windows':
-            mod_path = op.join(op.dirname(__file__), 'mod')
```

Comment:
this would just create inconsistency between Windows and other platforms. Better to build from the same directory and then find the .lib or .so file by recursive search.